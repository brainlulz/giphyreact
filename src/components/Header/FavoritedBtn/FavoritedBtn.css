.FavoritedBtn{
  grid-column: 1/1;
  grid-row:2/2;
  color: #FFBD39;
  justify-self: right;
  border: none;
  background: transparent;
  font-family: "SubZER0";
  font-size: 1.5rem;
  z-index: 1;
  width: 10rem;
}

.FavoritedBtn a {
  color: #FFBD39;
}

.FavoritedBtn .selected {
  color: #E61C5D;
}
