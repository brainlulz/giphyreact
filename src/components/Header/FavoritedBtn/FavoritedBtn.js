import React from 'react';
import { NavLink } from 'react-router-dom';
import './FavoritedBtn.css';

const Favorited = () => {
  return (
    <div className="FavoritedBtn">
      <NavLink to="/favorited" activeClassName="selected">
        Favorited
      </NavLink>
    </div>
  );
};

export default Favorited;
