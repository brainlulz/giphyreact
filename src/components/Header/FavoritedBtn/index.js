import FavoritedBtn from './FavoritedBtn';
export default FavoritedBtn;