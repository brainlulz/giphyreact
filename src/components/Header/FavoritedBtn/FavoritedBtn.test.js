import React from 'react';
import { shallow } from 'enzyme';
import FavoritedBtn from './index';

describe('FavoritedBtn', () => {
  it('renders the FavoritedBtn component', () => {
    const wrapper = shallow(<FavoritedBtn />);
    expect(wrapper).toMatchSnapshot();
  });
});
