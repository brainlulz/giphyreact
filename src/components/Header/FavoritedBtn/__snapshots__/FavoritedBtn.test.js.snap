// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoritedBtn renders the FavoritedBtn component 1`] = `
<button
  className="FavoritedBtn"
>
  <NavLink
    activeClassName="selected"
    ariaCurrent="true"
    to="/favorited"
  >
    Favorited
  </NavLink>
</button>
`;
